ext/ddtrace_profiling_native_extension/collectors_cpu_and_wall_time.c

```diff
@@ -91,7 +91,7 @@ struct cpu_and_wall_time_collector_state {
   // is not (just) a stat.
   unsigned int sample_count;
 
-  struct {
+  struct stats {
```

Comment:
This struct was previously anonymous, but I needed to name it so I could reference it from `_native_reset_after_fork`